lib/content/message/predictions.ex

```diff
@@ -151,11 +151,11 @@ defmodule Content.Message.Predictions do
           platform_name = Content.Utilities.stop_platform_name(stop_id)
 
           {headsign_message, platform_message} =
-            if minutes == :max_time,
-              do: {headsign, " (Platform TBD)"},
-              else:
-                {headsign <> " (#{String.slice(platform_name, 0..0)})",
-                 " (#{platform_name} plat)"}
+            unless minutes == :max_time or (is_integer(minutes) and minutes > 5) do
```

Comment:
nit: This logic looks right, but I find `unless` with `else` hard to read due to the double negative. Perhaps we should stick with `if`, and maybe invert the logic if it's preferable to have the typical case first.

Reply:
That's fair, I agree that `unless` can be difficult to read at a first pass. And I'm cool with leaving the logic as is to keep things simple!